src/App.js

```diff
@@ -0,0 +1,49 @@
+import "./App.css";
+import Switcher from "./Switcher";
+import Button from "./Button";
+import LandingPageHeader from "./LandingPageHeader";
+import ChangeCityModal from "./ChangeCityModal";
+import ModalCornerLink from "./ModalCornerLink";
+import SearchResultPage from "./SearchResultPage";
```

Comment:
Custom components should go in a components folder. Makes it easier to read and clearer when looking at the imports.

Reply:
Good point - made the requested change.